src/components/content/content.stories.mdx

```diff
@@ -10,36 +10,28 @@ contrast ratio of at least 4.5.1, to meet the [WCAG 2.0 AA standard](https://www
 for text less than 19px in size. Webaim have a good online [Contrast Checker](http://webaim.org/resources/contrastchecker/).
 
 ## Quick Start
-
-```jsx
+```js 
 import Content from "carbon-react/lib/components/content";
 ```
 
-## Default Content
+## Default content
 <Preview>
   <Story name="default">
-    <Content title='Title' as='primary'>
-      This is an example of some content
-    </Content>
+    <Content title='Title' variant='primary' children='This is an example of some content'/>
   </Story>
 </Preview>
 
-## Inline Content
+## Inline content
 <Preview>
-  <Story name="with Inline">
-    <Content title='Title' inline>
-      This is an example of some content
-    </Content>
+  <Story name="inline">
+    <Content title='Title' inline children='This is an example of some content'/>
```

Comment:
This is a strange use of the `children` prop - why did you change that from just having the text as children normally?

Reply:
wait.... what is it :D Let me check it, I don't remember I did that